qe/drivers.py

```diff
@@ -3476,7 +2836,7 @@ def local_sort_pt2_energies(
             # TODO: Maybe a bit hacky, but working fix for now... Argpartition throws error if no dets are generated in this constraint
             working_energies = np.r_[1, local_best_energies]
             # Add dummy
-            working_dets = [Determinant(alpha=(), beta=())] + local_best_dets
+            working_dets = local_best_dets + [Determinant(alpha=(), beta=())]
```

Comment:
Why we need to add a dummy? If it's to avoid empty maybe the 

```
working_dets =  local_best_dets
if !working_dets
   working_dets = [Determinant(alpha=(), beta=())]
```   
   if better